saucerbot/handlers.py

```diff
@@ -176,6 +176,7 @@ def dont_at_me(message, match):
 @app.handler(r'@ saucerbot')
 def sneaky(message, match):
     app.bot.post("you think you're sneaky don't you")
+    return True
```

Comment:
can you remove this `return True`?  I had changed it so that `@app.handler()`s that take an argument don't need a return statement